packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -77,6 +78,11 @@ class SnackBarThemeData extends Diagnosticable {
   /// otherwise.
   final Color disabledActionTextColor;
 
+  /// Used to configure the [DefaultTextStyle] for the [SnackBar.content] widget.
+  ///
+  /// If null, [SnackBar] defaults to a dark theme variant of [ThemeData.textTheme.subhead].
```

Comment:
I'd prefer not to document the fact that we're using the app's dark theme here  (except that we're not, and #31418). Although this PR just keeps things as they are, this can't be where we end up.

Reply:
Good call, removed